composer.json

```
@@ -18,7 +18,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "5.0.x-dev"
+            "dev-master": "6.0.x-dev"
```

what is this?

---

Makes the `master` branch available as `6.0.x-dev` as well.

---

OK, but what is the context? And sorry, if that's a stupid question

---

Basically it is used to tell what versions are cut from a branch. While that is clear on `1.0` or `5.3`, it isn't so much on `master`.

That being said: Why did you raise this here, @albe? The current versions for `neos/setup` are 4.x, so 5.x seems fine, if it is to be expected that 5.x will be created on `master` soonish. But 6.x means skipping one… 🤔

---

yes, that's what I meant with context :)

---

I might have been a bit to eager on this one :)
```suggestion
            "dev-master": "5.0.x-dev"
```